tests/accidental_tests.ts

```diff
@@ -163,9 +163,9 @@ function formatAccidentalSpaces(options: TestOptions): void {
       duration: '8d',
     }).addModifier(new Accidental('##'), 0),
     new StaveNote({
-      keys: ['b/4'],
+      keys: ['Bb/4','Bn/4'],
       duration: '16',
-    }).addModifier(new Accidental('b'), 0),
+    }).addModifier(new Accidental('b'), 0).addModifier(new Accidental('n'), 0),
```

Comment:
This is great!  Thank you!  I wonder if we should start as a good habit, adding in comments that explain the purpose of each test so that in the future if they're refactored, people know what is happening.  For instance, just adding here:

```javascript
\\ test that unisons with different accidentals layout properly with stems down
```

and later for D/4:

```javascript
\\ test that unisons with different accidentals layout properly with stems up
```

this way anyone who comes along later knows that replacing `Bb/4, Bn/4` with `Ab/4, An/4` isn't an equivalent test.

Reply:
(oops, I guess it is since there's a beamed note.  :-)